system/system_error_monitor/src/system_error_monitor_core.cpp

```diff
@@ -374,6 +375,11 @@ void AutowareErrorMonitor::onAutowareState(
 void AutowareErrorMonitor::onControlMode(
   const autoware_auto_vehicle_msgs::msg::ControlModeReport::ConstSharedPtr msg)
 {
+  // used for indicating that onControlMode callback is trigged at least once.
```

Comment:
```suggestion
  // used for indicating that onControlMode callback is called at least once.
```
Small typo on the word "triggered" but I think called is ok to use here.

Reply:
Fixed